File: src/main/java/com/example/demo/entity/UserBean.java
package com.example.demo.entity;

import javax.persistence.*;


@Entity
@Table(name = "user")
public class UserBean {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long user_id;

    @Column()
    private String user_name ;

    @Column()
    private String user_avatar;

    @Column(nullable = false)
    private String user_phone;

    @Column(nullable = false)
    private String user_password;

    public String getUser_name() {
        return user_name;
    }

    public void setUser_name(String user_name) {
        this.user_name = user_name;
    }

    public Long getUser_id() {
        return user_id;
    }

    public void setUser_id(Long user_id) {
        this.user_id = user_id;
    }

    public String getUser_avatar() {
        return user_avatar;
    }

    public void setUser_avatar(String user_avatar) {
        this.user_avatar = user_avatar;
    }

    public String getUser_phone() {
        return user_phone;
    }

    public void setUser_phone(String user_phone) {
        this.user_phone = user_phone;
    }

    public String getUser_password() {
        return user_password;
    }

    public void setUser_password(String user_password) {
        this.user_password = user_password;
    }
}

File: src/main/java/com/example/demo/entity/UserBean.kt
package com.example.demo.entity

import javax.persistence.*

/**
 * @author Mr_wu
 */
@Entity
@Table(name = "user")
class UserBean {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var user_id: Long? = null
    @Column
    var user_name: String? = null
    @Column
    var user_avatar: String? = null
    @Column(nullable = false)
    var user_phone: String? = null
    @Column(nullable = false)
    var user_password: String? = null

}